test/test-ffmpegCommandService.js

```diff
@@ -525,10 +693,63 @@ describe('FfmpegCommandService', function() {
+                    workingDirectory: _path('./fixtures/assets1')
+                });
+
+                // Glad we have whipple for this...
+                const expected = '/usr/local/bin/ffmpeg ' +
+                    `-i ${_path('./fixtures/assets1/sample.mp4')} ` +
+                    `-i ${_path('./fixtures/assets1/sample2.mp4')} ` +
+                    '-f lavfi -i color=c=555555:size=1920x1080 ' +
+                    '-f lavfi -i color=c=ff0000:size=1920x1080 ' +
+                    '-loop 1 ' +
+                    `-i ${_path('./fixtures/assets1/logo.png')} ` +
+                    '-loop 1 ' +
+                    `-i ${_path('./fixtures/assets1/logo.png')} ` +
+                    `-i ${_path('./fixtures/assets1/music.mp3')} ` +
+                    `-i ${_path('./fixtures/assets1/music2.mp3')} ` +
+                    '-filter_complex "' +
```

Comment:
As the commands got bigger I tried to use indentation to help clarify what was going on. Hopefully this makes it easier for the next person to edit.

Reply:
Thanks!